databases/week1/homework.sql

```diff
@@ -0,0 +1,39 @@
+/* 1. Find out how many tasks are in the task table */
+SELECT COUNT(*) AS task_count FROM task;
+
+/* 2. Find out how many tasks in the task table do not have a valid due date */
+SELECT COUNT(*) AS task_count_due_date_null FROM task WHERE due_date IS NULL;
+
+/* 3. Find all the tasks that are marked as done */
+SELECT * FROM task JOIN status ON task.status_id = status.id
```

Comment:
General advice: here, instead of selecting all columns i would select only the title and status name, it's enough to see the status of each task and the table doesn't get super wide :)

Reply:
You are correct, it would look better, I changed it :)